crates/deno-subsystem/deno-model-builder/src/system_builder.rs

```diff
@@ -114,7 +119,7 @@ fn process_script(
     // `BoxFuture`, which has the `Send` requirement.
     // TODO: Note that ProcState no longer exists in later versions of deno and has been replaced
     // with CliFactory.
```

Comment:
Not strictly related to this PR, but this comment is no longer relevant so can probably be removed.